backend/typescript/middlewares/validators/adminValidators.ts

```diff
@@ -60,3 +60,39 @@ export const formTemplateAddQuestionValidator = async (
   }
   return next();
 };
+
+export const formTemplateRemoveQuestionValidator = async (
+  req: Request,
+  res: Response,
+  next: NextFunction,
+) => {
+  if (
+    !req.params.formQuestionId ||
+    !validatePrimitive(req.params.formQuestionId, "string")
+  ) {
+    return res.status(400).send("formQuestionId param must be a string");
+  }
+  return next();
+};
+
+export const formTemplateEditQuestionValidator = async (
+  req: Request,
+  res: Response,
+  next: NextFunction,
+) => {
+  if (
+    !req.params.oldQuestionId ||
+    !validatePrimitive(req.params.oldQuestionId, "string")
+  ) {
+    return res.status(400).send("oldQuestionId param must be a string");
+  }
+
+  if (
+    !req.body.newFormQuestion ||
+    !validateFormQuestion(req.body.newFormQuestion)
```

Comment:
For editing questions, do users have to edit every field at once? What if they want to edit one field?

Reply:
The rest of the fields would remain the same as the previous data. It's not super restful but we are technically not PATCHing so I thought it was fine?

Reply:
Yeah I mean I'm not especially strongly opinionated here; I can see why in our use case it's easiest to just send the entire question back